app/controllers/application_controller.rb

```diff
@@ -1,2 +1,5 @@
 class ApplicationController < ActionController::Base
+  def current_user
+    User.last
+  end
```

Comment:
- [ ] The `current_user` method should return the first user from the database according to the project requirement. Please implement this change 👍